WordPress/Classes/System/WordPressAppDelegate.m

```diff
@@ -998,6 +1027,15 @@ - (void)removeTodayWidgetConfiguration
     [service removeTodayWidgetConfiguration];
 }
 
+#pragma mark - Simperium helpers
+
+- (void)resetSimperiumOnAuthTokenIssue
```

Comment:
Maybe leave a note on here for people to not call this and that this was specifically made for a hotfix?